src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -2,10 +2,14 @@ import React, { FC, useEffect, useState } from "react";
 import { Form, Input, Row, Col, useNotify } from "@canonical/react-components";
 import { FormikProps } from "formik";
 import StoragePoolFormMain from "./StoragePoolFormMain";
-import StoragePoolFormMenu from "./StoragePoolFormMenu";
+import StoragePoolFormMenu, { CEPH_CONFIGURATION } from "./StoragePoolFormMenu";
 import { MAIN_CONFIGURATION } from "./StorageVolumeFormMenu";
```

Comment:
This is the wrong import, it should be also taken from `StoragePoolFromMenu`. This is a previous mistake.

Reply:
good spot, changed